README.md

```diff
@@ -6,6 +6,8 @@
 
 ## Prerequisites
 
+1. **Subscription**. In order to use the OpenShift on Azure plugin, your Azure subscription must be whitelisted to use the service. If you do not use a valid subscription, you will not have access to the appropriate resource groups in order to successfully run code in this repository. Reach out to the #aos-azure team in [Slack][slack] with any questions about this.
```

Comment:
nit: let's avoid overly large lines and split them to fit the rest of the document

Reply:
I would love to truncate lines to 80 characters, I kept this as one long line to be consistent with the rest of the document :smile:

Do you want me to go ahead and reformat all of it?

Reply:
oh! huh the rest of the diff in here shows 80ish long lines... I guess whatever makes it more consistent with the rest of the doc :man_shrugging: if that's long lines, then disregard my comment.

Reply:
@ehashman I confess I have looked for a lightweight, safe and painless auto-formatting tool that we could standardise all our docs files with; worse I have failed.  I strongly prefer 80 character lines too.

Reply:
(not asking you to reformat this doc now though)